content/zh-cn/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -18,15 +18,15 @@ weight: 90
 <!--
 The Kubernetes API server is the main point of entry to a cluster for external parties
 (users and services) interacting with it.
- -->
+-->
```

Comment:
I think we don't need this kind of change, how about to sync security-checklist.md only?